src/eval-frame/actions/eval-frame-tasks.js

```diff
@@ -34,7 +34,7 @@ tasks.closePanes = new UserTask({
   callback() {
     postActionToEditor({
       type: 'CHANGE_SIDE_PANE_MODE',
-      sidePaneMode: '_CLOSED',
+      mode: '_CLOSED',
```

Comment:
hmm, tbh, i think it would be better to change the "CHANGE_SIDE_PANE_MODE" reducer in `src/eval-frame/reducers/eval-frame-reducer.js` and `/src/reducers/notebook-reducer.js` to look for `action.sidePaneMode` rather than changing this to pass "mode" since "mode" is not very descriptive and is already used elsewhere to mean the global mode

Reply:
Makes sense, I was thinking that myself while working on this, but wasn't sure. I'll fix things up tomorrow.